pkg/tests/schema_pulumi_test.go

```diff
@@ -2272,29 +2330,31 @@ Resources:
 Resources:
     2 unchanged
 `),
+			false,
 		},
 		// TODO[pulumi/pulumi-terraform-bridge#2235]: Wrong number of additions
-		// TODO[pulumi/pulumi-terraform-bridge#2325]: Non-deterministic output
-		// 		{
-		// 			"set element added front",
-		// 			map[string]interface{}{"setProps": []interface{}{"val2", "val3"}},
-		// 			map[string]interface{}{"setProps": []interface{}{"val1", "val2", "val3"}},
-		// 			autogold.Expect(`Previewing update (test):
-		//   pulumi:pulumi:Stack: (same)
-		//     [urn=urn:pulumi:test::test::pulumi:pulumi:Stack::test-test]
-		//     ~ prov:index/test:Test: (update)
-		//         [id=newid]
-		//         [urn=urn:pulumi:test::test::prov:index/test:Test::mainRes]
-		//       ~ setProps: [
-		//           ~ [0]: "val2" => "val1"
-		//           ~ [1]: "val3" => "val2"
-		//           + [2]: "val3"
-		//         ]
-		// Resources:
-		//     ~ 1 to update
-		//     1 unchanged
-		// `),
-		// 		},
+		{
+			"set element added front",
+			map[string]interface{}{"setProps": []interface{}{"val2", "val3"}},
+			map[string]interface{}{"setProps": []interface{}{"val1", "val2", "val3"}},
+			autogold.Expect(`Previewing update (test):
+		  pulumi:pulumi:Stack: (same)
```

Comment:
Nice, autogodl expect actual Pulumi output is highly trustworthy

Reply:
Given that the contract for the detailed diff response between the engine and the provider is not very trustworthy, this seems like the best thing we can do.